modules/Cargo.toml

```
@@ -19,7 +19,7 @@ tendermint = "0.15.0"
 tendermint-rpc = { version = "0.15.0", features = ["client"] }
 
 # Proto definitions for all IBC-related interfaces, e.g., connections or channels.
-ibc-proto = "0.3.0"
+ibc-proto = { path = "../proto" }
```

We need to specify a version number as well, otherwise we won't be able to publish the crate.

```suggestion
ibc-proto = { version = "0.3.0", path = "../proto" }
```